server/core/runtime/plan_step_runner.go

```diff
@@ -100,59 +104,7 @@ func (p *PlanStepRunner) remotePlan(ctx command.ProjectContext, extraArgs []stri
 	return p.fmtPlanOutput(output, tfVersion), nil
 }
 
-// switchWorkspace changes the terraform workspace if necessary and will create
-// it if it doesn't exist. It handles differences between versions.
-func (p *PlanStepRunner) switchWorkspace(ctx command.ProjectContext, path string, tfVersion *version.Version, envs map[string]string) error {
```

Comment:
this logic is moved into `workspace_step_runner_delegate.go` (`_delegate.go` is used like  minimum_version_step_runner_delegate.go)